examples/expositional/frameworks/llama_index/llama_index_groundtruth.ipynb

```diff
@@ -41,8 +41,9 @@
     "from llama_index.readers.web import SimpleWebPageReader\n",
     "import openai\n",
     "\n",
-    "from trulens_eval import TruLlama, Feedback, Tru, feedback\n",
-    "from trulens_eval.feedback import GroundTruthAgreement\n",
+    "from trulens.core import Tru, Feedback\n",
+    "from trulens.llamaindex import TruLlama\n",
+    "from trulens.external import OpenAI, GroundTruthAgreement\n",
```

Comment:
Suggesting to do a scheme like: trulens.feedback.openai
trulens.feedback.litellm
...